Automattic/woocommerce-connect-client
# Show errors inline and prevent form submit when errors present.

Requires https://github.com/Automattic/woocommerce-connect-server/pull/273 to test.

Fixes #206 and incorporates some error UI design from #306.

To test:
* Erase a required field value (like Title or Origin)
* Verify the error message is displayed
* Verify the "Save changes" button is disabled
* Enter an invalid zip code for Origin and
* Verify the error message is displayed
* Verify the "Save changes" button is disabled
* Add a package with only a name
* Verify that the package list item shows an error state
* Verify the "Save changes" button is disabled
* Force a bad value for a shipping service adjustment in the database by changing the serialized `i:0` value to `s:1:"a"`
* Reload form
* Verify that the group containing that service is expanded, and the service adjustment is highlighted red with an icon
